packages/swap/src/handlers/openSwapDepositChannel.ts

```diff
@@ -121,6 +121,7 @@ export default async function openSwapDepositChannel(
       quotedEgressAmount: input.quote.egressAmount,
       quotedPrice: input.quote.estimatedPrice,
       slippageTolerancePercent: getSlippageTolerancePercent(input),
+      recommendedSlippageTolerancePercent: input.quote.recommendedSlippageTolerancePercent,
```

Comment:
we also need to add a new property to `enabledFeatures` of the sdk class to request the auto slippage

Reply:
could call it `experimentalRecommendedSlippage`

Reply:
but can also do in a separate pr 🙂

Reply:
added it in this PR, would you please check again? :)